addons/eventbus/src/main/java/org/vaadin/spring/events/Event.java

```diff
@@ -31,14 +31,17 @@
 
     private final Object source;
 
+    private final String target;
+    
     private final long timestamp;
 
     private final T payload;
 
-    public Event(EventBus eventBus, Object source, T payload) {
+    public Event(EventBus eventBus, Object source, T payload, String target) {
```

Comment:
I'd like the Event to also have a constructor without a target that defaults to the empty string. Also, if the target is null, the constructor should convert it to an empty string.